lib/go-atscfg/headerrewritedotconfig.go

```diff
@@ -32,6 +34,8 @@ import (
 const HeaderRewritePrefix = "hdr_rw_"
 const ContentTypeHeaderRewriteDotConfig = ContentTypeTextASCII
 
+const ServiceCategoryHeader = "X-CDN-SVC"
```

Comment:
Would you mind changing this to `CDN-SVC`? The IETF has asked that people stop using `X-` for nonstandard headers, because they often become widespread, and  then it's difficult to make a new standard without the `X-`. Where if people don't  use `X-` to begin with, it's easy to make the header a standard when it becomes widespread.

Reply:
Source: https://tools.ietf.org/html/rfc6648